gen/dcos-config.yaml

```diff
@@ -1387,10 +1387,11 @@ package:
 {% case "true" %}
       MESOS_HTTP_COMMAND_EXECUTOR=true
       MESOS_HTTP_EXECUTOR_DOMAIN_SOCKETS=true
-      # TODO(bevers): With systemd versions before 227, named sockets are not supported and
-      # all socket names are rendered as "unknown". Change this to a named socket (i.e.
-      # `systemd:mesos-agent.socket`) once we switch to  more modern platforms.
-      MESOS_DOMAIN_SOCKET_LOCATION=systemd:unknown
+      # This variable is taken from the `FileDescriptorName` field from the unit
+      # file describing socket name of the Mesos agent service. Note that with systemd
+      # versions before 227, named sockets are not supported. However, Mesos agent
+      # handles this case.
+      MESOS_DOMAIN_SOCKET_LOCATION=systemd:dcos-mesos-slave
```

Comment:
where do we use `dcos-mesos-slave-public` socket name?

Reply:
I think this socket for a public agent service was added by this patch:
https://github.com/dcos/dcos/commit/48bde3c13558c037838a3a01727fb9ce9fb2fded#diff-1192067dec20d10423502fbb45588a81

Since we have 2 unit files describing public and private agent services, there are 2 corresponding `.socket` unit files.

Reply:
Please help me understand this - this file (`mesos-slave-common`) will be used by both public and private agents. WE configure the name to match private agents. Where do we configure `MESOS_DOMAIN_SOCKET_LOCATION` for public agents? Do we need to configure it?

Reply:
Ahhh... I see - the name is the same for both public and private agents :)